src/client/src/App.js

```diff
@@ -1,5 +1,5 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
+import logo from './logo.png';
```

Comment:
Love the new logo!

If you don't mind taking it an extra step, would be great to change the favicon and page title too: 
![image](https://user-images.githubusercontent.com/5824989/56749911-81cbf380-6737-11e9-868b-5d905898ba64.png)